Library/Homebrew/utils/analytics.rb

```diff
@@ -172,20 +172,24 @@ def get_analytics(json)
             end
           end
 
-          puts "#{category}: #{analytics.join(", ")}" unless full_analytics
+          analytics_categories.append("#{category}: #{analytics.join(", ")}") unless full_analytics
         end
+        output = ohai_title("Analytics") + "\n"
+        output << analytics_categories.join("\n")
+        output
```

Comment:
```suggestion
        [ohai_title("Analytics"), *analytics_categories].join("\n")
```

Does this need to guard for empty `analytics_categories`?